### DISCUSSION TITLE
Extracting audio tracks - choices of format

### DISCUSSION THREAD

**Discussion by cioeto44** [mifi/lossless-cut#1601] [Q&A]
Whith the new version downloaded yesterday (09.05.2023) when extractiing audio from an mp4 video file I get the naked aac. My favorite player - foobar - is not exactly great with aac files. That is why I would love to have a choice of going back to the way things were until now: the aac was rendered automatically in an m4a container. Is there some place in the new version where I can change the settings of this output?

**Comment by mifi** [comment]
could you just disable all tracks except the aac track, then export the file using the normal export button (and select your output format, I believe it's called `ipod (m4a)`)
